src/mono/wasi/build/WasiApp.targets

```diff
@@ -327,6 +327,7 @@
       <_WasmNativeFileForLinking Include="$(WasiSysRoot)\lib\wasm32-wasi\libc++.a" />
       <_WasmNativeFileForLinking Include="$(WasiSysRoot)\lib\wasm32-wasi\libc++abi.a" />
 
+      <NativeFileReference Include="%(NativeLibary.Identity)" Condition="'%(NativeLibary.Identity)' != ''"/>
```

Comment:
Just curious, can identity be empty? How would it be created?

Reply:
Updated this part.